Libraries/VaahPayPal.md

```diff
@@ -0,0 +1,77 @@
+# VaahPaypal
+> Small Laravel Helpers
+
+### Installation
+
+VaahPaypal provides to you a simple way to integrate Paypal in your Laravel application.
+
+### Dependencies
+- [PayPal-PHP-SDK](https://github.com/paypal/PayPal-PHP-SDK)
+
+### Installation
+
+```shell script
+composer require paypal/rest-api-sdk-php
+```
+
+[comment]: <> (Add Facade in `config/app.php`:)
```

Comment:
@we-himanshu-r001-v1 this is not visible at: https://github.com/webreinvent/vaahextend/blob/f2e591189b2221b16e2e5b748a130d079bceab51/Libraries/VaahPayPal.md

Reply:
same with other lines